### PR TITLE
Some code changes, need review.

### PR DESCRIPTION
**Pull request by cecekpawon**

### CODE REVIEW

File: Include/Acidanthera/Library/OcStringLib.h [hunk 1]
```diff
@@ -71,7 +71,7 @@ IsAsciiAlpha (
 
   @retval  TRUE, if character is a white space character.
 **/
-INTN
+BOOLEAN
```
Comment (PMheart) [inline Include/Acidanthera/Library/OcStringLib.h:74]: These are to match `isalpha` and `isspace` in `ctype.h`, aren't they?

File: Library/OcStringLib/OcAsciiLib.c [hunk 2]
```diff
@@ -57,7 +57,7 @@ IsAsciiAlpha (
 
   @retval  TRUE, if character is a white space character.
 **/
-INTN
+BOOLEAN
```
Comment (PMheart) [inline Library/OcStringLib/OcAsciiLib.c:60]: And same thing here.

File: Include/Acidanthera/Library/OcStringLib.h (hunk 1 above)
Comment (cecekpawon) [reply Include/Acidanthera/Library/OcStringLib.h:74]: Hello, sorry, im not sure here whether this function should match standard ctype or not. Im just refer to function header comment that retval shoud return TRUE if matched.

Comment (vit9696) [reply Include/Acidanthera/Library/OcStringLib.h:74]: Yeah, better fix the header then. Non-zero if character is a white space character, Zero otherwise.

**Comment by vit9696** [comment]
I cherry-picked the commits that make sense.

File: Include/Acidanthera/Library/OcStringLib.h (hunk 1 above)
Comment (cecekpawon) [reply Include/Acidanthera/Library/OcStringLib.h:74]: Thanks for review & merging @PMheart @vit9696 
I will leave this commit & close this pull req to let you guys do for the best regarding this commit.
- peace -

**Pull request closed without merging**